remix-tests/src/compiler.ts

```diff
@@ -21,6 +21,32 @@ function writeTestAccountsContract (accounts: string[]) {
     return testAccountContract.replace('>accounts<', body)
 }
 
+function processFile(filePath: string, sources: any, isRoot: boolean = false) {
+    try{
+        const importRegEx = /import ['"](.+?)['"];/g;
+        let group: any ='';
+        if(filePath.includes('tests.sol') || filePath.includes('remix_tests.sol') || filePath.includes('remix_accounts.sol') || Object.keys(sources).includes(filePath))
+            return
+
+        let content = fs.readFileSync(filePath, { encoding: 'utf-8' });
+        const s = /^(import)\s['"](remix_tests.sol|tests.sol)['"];/gm
+        if (isRoot && filePath.indexOf('_test.sol') > 0 && regexIndexOf(content, s) < 0) {
+            const includeTestLibs = '\nimport \'remix_tests.sol\';\n'
+            content = includeTestLibs.concat(content)
+        }
+        sources[filePath] = {content};
+        importRegEx.exec(''); // Resetting state of RegEx
+        while (group = importRegEx.exec(content)) {
+            const importedFile = group[1];
+            const importedFilePath = path.join(path.dirname(filePath), importedFile);
+            processFile(importedFilePath, sources)
+        }
+    }
+    catch(error){
```

Comment:
This is default behavior, you don't need to catch error if it's to throw it away just after.